lib/galaxy/jobs/runners/condor.py

```diff
@@ -246,3 +288,65 @@ def recover(self, job, job_wrapper):
             log.debug("(%s/%s) is still in DRM queued state, adding to the DRM queue" % (job.id, job.job_runner_external_id))
             cjs.running = False
             self.monitor_queue.put(cjs)
+
+
+
+    def _stop_container(self, job_wrapper):
+        return self._run_container_command(job_wrapper, 'stop')
+
+    def _kill_container(self, job_wrapper):
+        return self._run_container_command(job_wrapper, 'kill')
+
+    def _run_container_command(self, job_wrapper, command):
+        job = job_wrapper.get_job()
+        external_id = job.job_runner_external_id
+        if job:
+            cont = job.container
+            if cont:
+                if cont.container_type == 'docker':
+                    return self._run_command(cont.container_info['commands'][command], external_id)[0]
+
+    def _run_command(self, command, external_job_id):
+        command = 'condor_ssh_to_job %s %s' % (external_job_id, command)
```

Comment:
This should use the DockerCLI magic from the container utils already in Galaxy. At the current state, it's a proof of concept that this is possible for the condor runner.